src/main/java/com/opencart/pages/HeaderPage.java

```diff
@@ -48,6 +48,19 @@
     @FindBy(css = "button[onclick=\"wishlist.add('43');\"]")
     private WebElement macbookWishListButton;
 
+    @FindBy(css = "a[title = 'Shopping Cart'] span")
+    private WebElement shoppingCartSpan;
+
+    @FindBy(css = "span#cart-total")
+    private WebElement shoppingCartDropDownInfo;
+
+    @FindBy(css = "p.text-center")
+    private WebElement shoppingCartDropDownMessage;
+
```

Comment:
Before making the commit try to use shortcuts to format your code, so all the extra spaces will be removed automatically.